service/matching/physical_task_queue_manager.go

```diff
@@ -466,6 +467,22 @@ func (c *physicalTaskQueueManagerImpl) HasPollerAfter(accessTime time.Time) bool
 	return len(recentPollers) > 0
 }
 
+func (c *physicalTaskQueueManagerImpl) HasVersionedPollerAfter(accessTime time.Time, buildID string) bool {
```

Comment:
if not in this PR, later we should move out the versioned pollers into their own physicalTaskQueueManager so this method is not needed because we call `HasPollerAfter` on the right physicalTaskQueueManager, not on the defaultQueue all the time.

Reply:
I started working on that here https://github.com/temporalio/temporal/pull/5501/

Reply:
It doesn't work yet, whereas `HasVersionedPollerAfter` does, so I feel like we could merge this and then make #5501 work